hack/get-rosa-docs.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
Looks good! I do have a minor suggestion, though. Would it be possible to include a comment that explains the purpose of the document?  This would provide a helpful overview of its function for future reference.

Reply:
Good point, thanks.

I added a few more comments now. Hopefully it is more clear now?

Reply:
Thanks @codificat . 

/lgtm

Reply:
/lgtm